bundle/language_en.properties

```diff
@@ -662,6 +663,21 @@ angal.hospital.visitduration.txt
 angal.hospital.visitendhour.txt                                                                        = Visit end hour
 angal.hospital.visitstarthour.txt                                                                      = Visit start hour
 angal.lab.allnegative.txt                                                                              = All negative.
+angal.inventory.close.btn                                                                              = Close
+angal.inventory.date.col                                                                               = Date
+angal.inventory.delete.btn                                                                             = Delete inventory
```

Comment:
And here: **angal.common.delete.btn**.     I am not going to go through the whole list.